ArduCopter/system.cpp

```diff
@@ -327,9 +324,6 @@ bool Copter::ekf_position_ok() const
 // optflow_position_ok - returns true if optical flow based position estimate is ok
 bool Copter::optflow_position_ok() const
 {
-#if OPTFLOW != ENABLED && VISUAL_ODOMETRY_ENABLED != ENABLED
```

Comment:
You chose to simplify this, I see.  This could have changed to use the new `HAL_` define.

Reply:
yes, and I considered this but I couldn't see any reason to have two different ways to arrive at the same conclusion so I removed the short-cut version.